libs/feature/search/src/lib/filter-dropdown/filter-dropdown.component.ts

```diff
@@ -33,13 +33,8 @@ export class FilterDropdownComponent implements OnInit {
     startWith([])
   )
   selected$ = this.searchFacade.searchFilters$.pipe(
-    map(
-      (filters) =>
-        filters[this.fieldName] && Object.keys(filters[this.fieldName])
-    ),
-    filter((selected) => !!selected),
-    take(1),
```

Comment:
The stream was crap and was not working at all actually. It was just doing something on page loading and just closed after init because of the `take(1)`
What is means also is that mart `selected$` is useless for internal purposes, i.e when the filters change because of multiple dropdown update. 
Actually, the dump component is stateful, which might not be appropriate @jahow